pages/index.js

```diff
@@ -39,15 +39,15 @@ export default function Home() {
               NeoHacks
             </h1>
             <h3 className="w-2/3 text-xl text-white font-fira md:text-2xl md:w-full">
-              August 6, 2022 &middot; Hybrid
+              August 6-7th, 2022 &middot; Hybrid
             </h3>
 
             <motion.button
               whileHover={{ scale: 1.1 }}
               whileTap={{ scale: 0.9 }}
               className="p-2 px-6 text-base font-medium bg-white rounded-full font-inter md:text-lg w-min"
             >
-              <Link href="/register">Register</Link>
+              <Link href="https://airtable.com/shrshwlldsgdG9GVs">Register</Link>
```

Comment:
```suggestion
              <Link href="/register">Register</Link>
```

See next.config.js; it is a redirect to the Airtable link.

Reply:
will do